python/tk_multi_loader/dialog.py

```diff
@@ -154,15 +181,14 @@ def __init__(self, action_manager, parent=None):
         # hook up view -> proxy model -> model
         self.ui.publish_view.setModel(self._publish_proxy_model)
 
-        # customize the list view before setting the delegate
-        self._add_rdo_list_view_mod()
+        # set up custom delegates to use when drawing the main area
+        self._publish_thumb_delegate = SgPublishThumbDelegate(self.ui.publish_view, self._action_manager)
 
-        # tell our publish view to use a custom delegate to produce widgetry
-        if self.ui.publish_view.ViewMode() == QtGui.QListView.IconMode:
-            self._publish_delegate = SgPublishDelegate(self.ui.publish_view, self._status_model, self._action_manager)
-        else:
-            self._publish_delegate = SgPublishListDelegate(self.ui.publish_view, self._status_model, self._action_manager)
-        self.ui.publish_view.setItemDelegate(self._publish_delegate)
+        self._publish_list_delegate = SgPublishListDelegate(self.ui.publish_view, self._action_manager)
+        
+        # recall which the most recently mode used was and set that
+        main_view_mode = self._settings_manager.retrieve("main_view_mode", self.MAIN_VIEW_THUMB)
```

Comment:
In https://github.com/rodeofx/tk-multi-loader2/pull/1, Julien added a "display_thumbnails" setting to the info.yml to determine which of the thumbnail view or list view should be used.
This setting should be used as the default here instead of "self.MAIN_VIEW_THUMB"

```
        default_view_mode = self.MAIN_VIEW_THUMB
        if not sgtk.platform.current_bundle().get_setting("display_thumbnails"):
            default_view_mode = self.MAIN_VIEW_LIST
        main_view_mode = self._settings_manager.retrieve("main_view_mode", default_view_mode)
```